setup.py

```diff
@@ -341,11 +341,19 @@ def getExtensionModules(nupicCoreReleaseDir, platform, bitness):
   commonLinkFlags = [
     "-m" + bitness,
     "-fPIC",
-    "-L" + pythonPrefix + "/lib"]
+    "-L" + pythonPrefix + "/lib",
+    "-L" + nupicCoreReleaseDir + "/lib",
```

Comment:
Just curious, why did you add this line? Does it have anything to do with the rest of the PR?